cmd/snap/notes.go

```diff
@@ -83,6 +86,7 @@ func NotesFromLocal(snap *client.Snap) *Notes {
 	return &Notes{
 		Private:  snap.Private,
 		DevMode:  snap.DevMode,
+		Classic:  snap.Confinement == client.ClassicConfinement,
```

Comment:
With the original code that was okay. With the revised devmode + classic it is not. Fixed, thanks for pointing this out.